### PR TITLE
Update index_en.rst

### PR DESCRIPTION
**Pull request by shanyi15**
fix https://github.com/PaddlePaddle/Paddle/issues/8921

### CODE REVIEW

File: doc/v2/faq/index_en.rst [hunk 1]
```diff
@@ -1,7 +1,8 @@
 FAQ
 ====
 
- 
+This document provides frequently asked questions of PaddlePaddle. If your questions are not here, please go to `PaddlePaddle Community <https://github.com/PaddlePaddle/Paddle/issues>`_ , to find answers or open an `issue <https://github.com/PaddlePaddle/Paddle/issues/new>`_  , we will reply in time.
+
```
Comment (chengduoZH) [inline doc/v2/faq/index_en.rst]: 1. This document provides answers to some of the frequently asked questions but not only questions.
`This document provides frequently asked questions of PaddlePaddle.`
==>
`This document provides answers to some of the frequently asked questions about PaddlePaddle.`

2. `If your questions are not here` 
==>
`If you have a question that not covered here`

Comment (shanyi15) [reply doc/v2/faq/index_en.rst]: Done, covered是被动语态，感觉修改意见中缺一个be动词，遂考虑在that和not中间加一个is。

Comment (shanyi15) [reply doc/v2/faq/index_en.rst]: Reference:
image

File: doc/v2/faq/index_en.rst [hunk 2]
```diff
@@ -1,7 +1,8 @@
 FAQ
 ====
 
- 
+This document provides answers to some of the frequently asked questions about PaddlePaddle. If you have a question that is not covered here, please go to `PaddlePaddle Community <https://github.com/PaddlePaddle/Paddle/issues>`_ , to find answers or open an `issue <https://github.com/PaddlePaddle/Paddle/issues/new>`_  , we will reply in time.
+
```
Comment (chengduoZH) [inline doc/v2/faq/index_en.rst]: 1. `find answers` ==> `find an answer`
2. How about replacing `open an issue` with `submit new issue`?

Comment (shanyi15) [reply doc/v2/faq/index_en.rst]: Done

**Pull request merged**